pkg/deployment/collector.go

```diff
@@ -76,6 +78,16 @@ func (c *Collector) Get() *appsv1.Deployment {
 	sampling.Update(c.jaeger, commonSpec, &options)
 	tls.Update(c.jaeger, commonSpec, &options)
 
+	otelConfg, err := c.jaeger.Spec.Collector.Config.GetMap()
+	if err != nil {
+		c.jaeger.Logger().WithField("error", err).
```

Comment:
add `component`, as if other components also have otelconfig would be hard to distinguish. (same comment for other components).